front/app/components/Mounter/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react';
```

Comment:
`OnMounted`  makes it sounds like a callback function, but it's a component

Reply:
Mounter otoh makes it sound as if it's a component that helps something mount 😁 Not very deep into this but isn't it possible to add it as a prop to the Outlet instead and call the useEffect there?

And/or add it as a useEffect to the components you're rendering in the outlet like we do with onData in tab components?

Anyway, if this works, we can also take it out of this PR and discuss in the team-front channel. No need to delay the release because of this. 🙂

Reply:
> isn't it possible to add it as a prop to the Outlet instead and call the useEffect there?
That would work, but only if we can call `onMount` after the JSX is rendered, which would require a `Mounter`-like component inside the `Outlet` . But might be cleaner indeed

> And/or add it as a useEffect to the components you're rendering in the outlet like we do with onData in tab components?
Already doing something like this with `onDataLoaded`, but you still have to be sure that `onDataLoaded` was called with `false` first... and you don't know beforehand how many outlets are going to be rendered

> Anyway, if this works, we can also take it out of this PR and discuss in the team-front channel. No need to delay the release because of this.
Yes will bring this up later when I work on this again :+1: